src/cellfunction/cellfunction.jl

```diff
@@ -614,6 +527,44 @@ function LazyOperators.materialize(::CellNormal, cPoint::CellPoint{ReferenceDoma
     return cell_normal(ctype, cnodes, ξ)
 end
 
+function LazyOperators.materialize(
+    ν::CellNormal,
+    ::AbstractSide{Nothing, <:Tuple{<:FaceInfo}},
+)
+    ν
+end
+
+function LazyOperators.materialize(
+    ::CellNormal,
+    sideFacePoint::Side⁻{Nothing, <:Tuple{<:FacePoint}},
+)
+    fPoint, = get_args(sideFacePoint)
+    fInfo = get_faceinfo(fPoint)
+    cInfo = get_cellinfo_n(fInfo)
+    cnodes = nodes(cInfo)
+    ctype = celltype(cInfo)
+    ξcell = get_coords(side_n(fPoint))
+
+    return cell_normal(ctype, cnodes, ξcell)
+end
+
+function LazyOperators.materialize(
+    ::CellNormal,
+    sideFacePoint::Side⁺{Nothing, <:Tuple{<:FacePoint}},
+)
+    fPoint, = get_args(sideFacePoint)
+    fInfo = get_faceinfo(fPoint)
+    cInfo = get_cellinfo_p(fInfo)
+    cnodes = nodes(cInfo)
+    ctype = celltype(cInfo)
+    ξcell = get_coords(side_p(fPoint))
+
+    return cell_normal(ctype, cnodes, ξcell)
+end
+
+_tangeantial_projector(ν) = I - (ν ⊗ ν)
```

Comment:
typo, should be "tangential" not "tangeantial"

Reply:
done